print_conversations.py

```diff
@@ -1,38 +1,51 @@
 import os
+import glob
+import os
```

Comment:
Can you fix the duplicate `os` import? Thanks!

Reply:
You can do this just by committing the fix to this branch and pushing it. The new commit will be incorporated in the PR

Reply:
Sounds good, done!